core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -513,6 +513,14 @@ final case class EitherT[F[_], A, B](value: F[Either[A, B]]) {
   def leftSemiflatTap[C](f: A => F[C])(implicit F: Monad[F]): EitherT[F, A, B] =
     leftSemiflatMap(a => F.as(f(a), a))
 
+  def biSemiflatTap[C, D](fa: A => F[C], fb: B => F[D])(implicit F: Monad[F]): EitherT[F, A, B] =
+    EitherT(F.flatMap(value) {
+      case l @ Left(a) =>
+        F.as(fa(a), l)
+      case r @ Right(b) =>
+        F.as(fb(b), r)
+    })
+
```

Comment:
Appears that it only uses `F.flatMap` and `F.as` (if I'm not missing something)
Both of them are accessible via `FlatMap` typeclass.
I wonder if it makes sense to loosen the constraint to `F: FlatMap[F]` instead of `Monad`.

Reply:
Done; rebased 393b5b86b6c47c6d25ad24bbe5ea6bbb5434f8af